Purchases/SwiftSources/LocalReceiptParsing/Builders/ASN1ObjectIdentifierBuilder.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Created by Andrés Boedo on 7/28/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+class ASN1ObjectIdentifierBuilder {
```

Comment:
I'll add docs for this one, the algorithm is simple to do but online documentation sucks.